eldritch/test_usables.py

```diff
@@ -374,6 +376,148 @@ def testCannotBeCastOutsideOtherworldMovement(self):
   # TODO: a test covering the ability to cast after travelling during the movement phase
 
 
+class FleshWardTest(EventTest):
+  def setUp(self):
+    super().setUp()
+    self.flesh_ward = items.FleshWard(0)
+    self.char.possessions.append(self.flesh_ward)
+    self.char.possessions.append(items.TommyGun(0))
+
+  def testCombat(self):
+    monster = monsters.Cultist()
+    combat = Combat(self.char, monster)
+    self.state.event_stack.append(combat)
+    fight_or_flee = self.resolve_to_choice(MultipleChoice)
+    fight_or_flee.resolve(self.state, "Fight")
+    combat_choice = self.resolve_to_choice(CombatChoice)
+    combat_choice.resolve(self.state, "done")
+    with mock.patch.object(events.random, "randint", new=mock.MagicMock(return_value=4)):
+      self.resolve_to_usable(0, "Flesh Ward0", events.CastSpell)
+    self.state.event_stack.append(self.state.usables[0]["Flesh Ward0"])
+    choice = self.resolve_to_choice(SpendMixin)
+    self.assertListEqual(choice.choices, ["Cast", "Cancel"])
+    with mock.patch.object(events.random, "randint", new=mock.MagicMock(return_value=5)):
+      self.spend("sanity", 1, choice)
+      choice.resolve(self.state, "Cast")
+      fight_or_flee = self.resolve_to_choice(MultipleChoice)
+
+    self.assertEqual(self.char.stamina, 5)
+    self.assertEqual(self.char.sanity, 4)
+
+    fight_or_flee.resolve(self.state, "Fight")
+    combat_choice = self.resolve_to_choice(CombatChoice)
+    combat_choice.resolve(self.state, "done")
+    with mock.patch.object(events.random, "randint", new=mock.MagicMock(return_value=4)):
+      self.resolve_to_choice(MultipleChoice)
+      # no longer usable
+      self.assertNotIn("Flesh Ward0", self.state.usables)
```

Comment:
Nit: self.state.usables is a dict that maps idx -> a dict from handle -> event. So actually, you want to check that `0` is not in self.state.usables here. Or just asserting that it's empty is enough (as you do on the line below).